hyperspy/axes.py

```diff
@@ -2009,87 +2029,78 @@ def update_axes_attributes_from(self, axes,
         if changes:
             self.events.any_axis_changed.trigger(obj=self)
 
-    def _update_attributes(self):
-        getitem_tuple = []
-        values = []
-        signal_axes = ()
-        navigation_axes = ()
+    @observe("_axes")
+    @observe(trait("_axes", notify=False).list_items())
```

Comment:
Does it require to bump `traits` minimum requirement to 6.1?

Reply:
Yes we should update that dependency